source/persistency/PetaloPersistencyManager.cc

```diff
@@ -285,18 +284,18 @@ void PetaloPersistencyManager::StoreIonizationHits(G4VHitsCollection* hc)
 
 void PetaloPersistencyManager::StoreSensorHits(G4VHitsCollection* hc)
 {
-  std::map<G4int, SensorHit*> mapOfHits;
+  std::map<G4int, PetSensorHit*> mapOfHits;
```

Comment:
`PetSensorHit ` is only for TOF?

Reply:
I wanted to use just one class for everything. Have a look at the class and tell me if something isn't clear.